src/zope/testrunner/tests/testrunner-ex-37/stop_on_failure.py

```diff
@@ -0,0 +1,31 @@
+##############################################################################
+#
+# Copyright (c) 2013 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+
+import unittest
+
+
+class FailureTestCase1(unittest.TestCase):
+    layer = "layers.LayerA"
+
+    def test(self):
+        # We want to have an error, not a failure
+        raise StandardError
```

Comment:
There's no "StandardError" on Python 3.

It doesn't really matter what sort of exception you raise, as long as it's not AssertionError -- I suggest `raise Exception`.